easybuild/easyconfigs/g/GTK+/GTK+-3.24.31-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,76 @@
+easyblock = 'Bundle'
+
+name = 'GTK+'
+version = '3.24.31'
+
+homepage = 'https://developer.gnome.org/gtk3/stable/'
+description = """GTK+ is the primary library used to construct user interfaces in GNOME. It
+ provides all the user interface controls, or widgets, used in a common
+ graphical application. Its object-oriented API allows you to construct
+ user interfaces without dealing with the low-level details of drawing and
+ device interaction.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+builddependencies = [
+    ('binutils', '2.37'),
+    ('Autotools', '20210726'),
+    ('GObject-Introspection', '1.68.0'),
+    ('gettext', '0.21'),
+    ('pkg-config', '0.29.2'),
+    ('cairo', '1.16.0'),
+    ('Perl', '5.34.0'),
+]
+dependencies = [
+    ('ATK', '2.36.0'),
+    ('at-spi2-atk', '2.38.0'),
+    ('Gdk-Pixbuf', '2.42.6'),
+    ('Pango', '1.48.8'),
+    ('libepoxy', '1.5.8'),
+    ('X11', '20210802'),
+    ('FriBidi', '1.0.10'),
+]
+
+default_easyblock = 'ConfigureMake'
+
+default_component_specs = {
+    'sources': [SOURCELOWER_TAR_XZ],
+    'start_dir': '%(namelower)s-%(version)s',
+}
+
+components = [
+    (name, version, {
```

Comment:
```suggestion
    ('GTK+', version, {
```

Reply:
Thanks - missed that at the top of my search for `^GTK`...